GirafRest/Services/GirafService.cs

```diff
@@ -77,6 +77,7 @@ public async Task<GirafUser> LoadUserWithDepartment(System.Security.Claims.Claim
             var usr = (await _userManager.GetUserAsync(principal));
             if (usr == null) return null;

```

Comment:
Jeg er med på, at det ikke er jer der har lavet den del, men er det ikke underligt at man tjekker for null og så bare returnere null? Så kunne man jo ligeså godt lade være med at tjekke det? :D